infra/build/Makefile

```diff
@@ -27,7 +27,7 @@ PACKER_VERSION := 1.4.4
 TERRAGRUNT_VERSION := 0.23.25
 KUSTOMIZE_VERSION := 3.6.0
 
-DOCKER_TAG_VERSION_DEVELOPER_TOOLS := 0.13.15
+DOCKER_TAG_VERSION_DEVELOPER_TOOLS := 0.14.0
```

Comment:
This can probably still be 0.13, no? Technically just a bugfix.

Reply:
sg, will update